<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="6" total="6" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="18519" started-at="2016-09-05T06:00:10Z" finished-at="2016-09-05T06:00:29Z">
    <groups>
    </groups>
    <test name="FirefoxTest" duration-ms="18519" started-at="2016-09-05T06:00:10Z" finished-at="2016-09-05T06:00:29Z">
      <class name="testscripts.Book_search">
        <test-method status="PASS" signature="launchApp(java.lang.String)[pri:0, instance:testscripts.Book_search@617faa95]" name="launchApp" is-config="true" duration-ms="10731" started-at="2016-09-05T11:30:10Z" finished-at="2016-09-05T11:30:21Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApp -->
        <test-method status="PASS" signature="setreport()[pri:0, instance:testscripts.Book_search@617faa95]" name="setreport" is-config="true" duration-ms="149" started-at="2016-09-05T11:30:21Z" finished-at="2016-09-05T11:30:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setreport -->
        <test-method status="FAIL" signature="valid(java.util.HashMap)[pri:0, instance:testscripts.Book_search@617faa95]" name="valid" duration-ms="24" started-at="2016-09-05T11:30:22Z" data-provider="bsearch" finished-at="2016-09-05T11:30:22Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Execute_Flag=Y, TC_ID=TC_01, ISBN=, BookSearch=Ways, Scriptname=Valid, Author=, Publisher=}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at PageFactory.PageFactory_Search.search_book(PageFactory_Search.java:40)
	at testscripts.Book_search.valid(Book_search.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- valid -->
        <test-method status="PASS" signature="CaptureScreenshot(org.testng.ITestResult)[pri:0, instance:testscripts.Book_search@617faa95]" name="CaptureScreenshot" is-config="true" duration-ms="907" started-at="2016-09-05T11:30:22Z" finished-at="2016-09-05T11:30:23Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=valid status=FAILURE method=Book_search.valid(java.util.HashMap)[pri:0, instance:testscripts.Book_search@617faa95] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CaptureScreenshot -->
        <test-method status="FAIL" signature="valid(java.util.HashMap)[pri:0, instance:testscripts.Book_search@617faa95]" name="valid" duration-ms="6" started-at="2016-09-05T11:30:23Z" data-provider="bsearch" finished-at="2016-09-05T11:30:23Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Execute_Flag=Y, TC_ID=TC_02, ISBN=, BookSearch=, Scriptname=Valid, Author=Chandler, Publisher=}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at PageFactory.PageFactory_Search.search_book(PageFactory_Search.java:40)
	at testscripts.Book_search.valid(Book_search.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- valid -->
        <test-method status="PASS" signature="CaptureScreenshot(org.testng.ITestResult)[pri:0, instance:testscripts.Book_search@617faa95]" name="CaptureScreenshot" is-config="true" duration-ms="925" started-at="2016-09-05T11:30:23Z" finished-at="2016-09-05T11:30:24Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=valid status=FAILURE method=Book_search.valid(java.util.HashMap)[pri:0, instance:testscripts.Book_search@617faa95] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CaptureScreenshot -->
        <test-method status="FAIL" signature="valid(java.util.HashMap)[pri:0, instance:testscripts.Book_search@617faa95]" name="valid" duration-ms="2" started-at="2016-09-05T11:30:24Z" data-provider="bsearch" finished-at="2016-09-05T11:30:24Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Execute_Flag=Y, TC_ID=TC_03, ISBN=, BookSearch=, Scriptname=Valid, Author=, Publisher=Mc}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at PageFactory.PageFactory_Search.search_book(PageFactory_Search.java:40)
	at testscripts.Book_search.valid(Book_search.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- valid -->
        <test-method status="PASS" signature="CaptureScreenshot(org.testng.ITestResult)[pri:0, instance:testscripts.Book_search@617faa95]" name="CaptureScreenshot" is-config="true" duration-ms="679" started-at="2016-09-05T11:30:24Z" finished-at="2016-09-05T11:30:24Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=valid status=FAILURE method=Book_search.valid(java.util.HashMap)[pri:0, instance:testscripts.Book_search@617faa95] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CaptureScreenshot -->
        <test-method status="FAIL" signature="valid(java.util.HashMap)[pri:0, instance:testscripts.Book_search@617faa95]" name="valid" duration-ms="2" started-at="2016-09-05T11:30:24Z" data-provider="bsearch" finished-at="2016-09-05T11:30:24Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Execute_Flag=Y, TC_ID=TC_04, ISBN=97808, BookSearch=, Scriptname=Valid, Author=, Publisher=}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at PageFactory.PageFactory_Search.search_book(PageFactory_Search.java:40)
	at testscripts.Book_search.valid(Book_search.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- valid -->
        <test-method status="PASS" signature="CaptureScreenshot(org.testng.ITestResult)[pri:0, instance:testscripts.Book_search@617faa95]" name="CaptureScreenshot" is-config="true" duration-ms="711" started-at="2016-09-05T11:30:24Z" finished-at="2016-09-05T11:30:25Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=valid status=FAILURE method=Book_search.valid(java.util.HashMap)[pri:0, instance:testscripts.Book_search@617faa95] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CaptureScreenshot -->
        <test-method status="FAIL" signature="valid(java.util.HashMap)[pri:0, instance:testscripts.Book_search@617faa95]" name="valid" duration-ms="1" started-at="2016-09-05T11:30:25Z" data-provider="bsearch" finished-at="2016-09-05T11:30:25Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Execute_Flag=Y, TC_ID=TC_05, ISBN=, BookSearch=abcd, Scriptname=Invalid, Author=, Publisher=}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at PageFactory.PageFactory_Search.search_book(PageFactory_Search.java:40)
	at testscripts.Book_search.valid(Book_search.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- valid -->
        <test-method status="PASS" signature="CaptureScreenshot(org.testng.ITestResult)[pri:0, instance:testscripts.Book_search@617faa95]" name="CaptureScreenshot" is-config="true" duration-ms="721" started-at="2016-09-05T11:30:25Z" finished-at="2016-09-05T11:30:26Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=valid status=FAILURE method=Book_search.valid(java.util.HashMap)[pri:0, instance:testscripts.Book_search@617faa95] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CaptureScreenshot -->
        <test-method status="FAIL" signature="valid(java.util.HashMap)[pri:0, instance:testscripts.Book_search@617faa95]" name="valid" duration-ms="1" started-at="2016-09-05T11:30:26Z" data-provider="bsearch" finished-at="2016-09-05T11:30:26Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Execute_Flag=N, TC_ID=TC_06, ISBN=, BookSearch=, Scriptname=Invalid, Author=PQRS, Publisher=}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at PageFactory.PageFactory_Search.search_book(PageFactory_Search.java:40)
	at testscripts.Book_search.valid(Book_search.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- valid -->
        <test-method status="PASS" signature="CaptureScreenshot(org.testng.ITestResult)[pri:0, instance:testscripts.Book_search@617faa95]" name="CaptureScreenshot" is-config="true" duration-ms="762" started-at="2016-09-05T11:30:26Z" finished-at="2016-09-05T11:30:27Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=valid status=FAILURE method=Book_search.valid(java.util.HashMap)[pri:0, instance:testscripts.Book_search@617faa95] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CaptureScreenshot -->
        <test-method status="PASS" signature="stopvideo()[pri:0, instance:testscripts.Book_search@617faa95]" name="stopvideo" is-config="true" duration-ms="2033" started-at="2016-09-05T11:30:27Z" finished-at="2016-09-05T11:30:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- stopvideo -->
        <test-method status="PASS" signature="endreport()[pri:0, instance:testscripts.Book_search@617faa95]" name="endreport" is-config="true" duration-ms="436" started-at="2016-09-05T11:30:29Z" finished-at="2016-09-05T11:30:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endreport -->
      </class> <!-- testscripts.Book_search -->
    </test> <!-- FirefoxTest -->
  </suite> <!-- Suite -->
</testng-results>
